README.md

```diff
@@ -40,13 +40,16 @@ terragrunt-atlantis-config generate
 terragrunt-atlantis-config generate --root /some/path/to/your/repo/root
 
 # output to a file
-terragrunt-atlantis-config generate --autoplan --output ./output.tf
+terragrunt-atlantis-config generate --autoplan --output ./atlantis.yaml
 
 # enable auto plan
 terragrunt-atlantis-config generate --autoplan
 
 # define the workflow
-terragrunt-atlantis-config generate --workflow web --output ./output.tf
+terragrunt-atlantis-config generate --workflow web --output ./atlantis.yaml
+
+# ignore terragrunt configs which do not reference a terraform module, thus, can't be planned
+terragrunt-atlantis-config generate --ignore-unplannable
```

Comment:
I'm a little concerned that this name suggests ignoring modules where a call to `terraform plan` fails. What would you think of changing the name of the flag to something like `--ignore-parent-modules`?

Reply:
Yeah I'm not thrilled with the naming here either 🤔. Although the term module seems to be reserved for pure-terraform stuff.  I got an idea though💡

Reply:
`--ignore-parent-terragrunt` looks good to me. Thanks!